src/ScottPlot5/ScottPlot/Control/Key.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ScottPlot.Control
+{
+    public enum Key // Handling all possible keys is non-trivial and probably unnecessary
```

Comment:
Let's not change the design now, but...

Rather than handling all keys I wonder if there's a good way to abstract this (rather than using an enum) so users can extend it themselves to add new or custom keys.

I use OriginLab (scientific software similar to MatLab) at work, and their zooming controls are nuts. Shift + X + drag to pan vertically 😝 

* https://www.originlab.com/doc/Origin-Help/Pan-GraphPage
* https://www.originlab.com/doc/Origin-Help/Zoom-GraphPage

Reply:
Simplest option is probably just to pass in a numeric ID (for example a scan code). That's a little low level, but I think if the issue is that we don't want to make a good abstraction, well then maybe we shouldn't cram the user into a bad one.

So then we do something like this:
```cs
public enum KnownKeys {
  Ctrl,
  Alt,
  Shift
};

public struct Key {
  public KnownKey? KnownKey { get; }
  public int ScanCode { get; }
  
  public Key(int scanCode) {
    ScanCode = scanCode;
    KnownKey = GetKeyFromScanCode(scanCode)
  }
}
```

FWIW Avalonia (and probably more) have lookup-tables for scancodes, if we wanted to provide something deeper in the future.